src/AutoRest.CSharp/Common/Generation/Types/TypeFactory.cs

```diff
@@ -74,6 +74,7 @@ public TypeFactory(OutputLibrary library)
                 InputTypeKind.Uri => new CSharpType(typeof(Uri), inputType.IsNullable),
                 _ => new CSharpType(typeof(object), inputType.IsNullable),
             },
+            InputIntrinsicType { Kind: InputIntrinsicTypeKind.Unknown } => typeof(BinaryData),
```

Comment:
We handle the unknow  type from typespec. Shall we convert `any` type from codeModel to Unknow type in CodeModelConvert?

Reply:
> We handle the unknow type from typespec. Shall we convert `any` type from codeModel to Unknow type in CodeModelConvert?

Yes